systemtests/src/test/java/io/enmasse/systemtest/isolated/plans/standard/PlansTestStandard.java

```diff
@@ -1168,6 +1168,88 @@ void testUnknownAddressPlan() throws Exception {
         doTestUnknownAddressPlan(addressSpace, stageHolders);
     }
 
+    @Test
+    void testUpdatePlanBrokerCredit_changesPerAddressMaxSize() throws Exception {
```

Comment:
same here please avoid using `_` and also add this test into acceptance profile then we can see behavior on ocp4